08_project/n0npax/pkg/puppy/errors.go

```diff
@@ -0,0 +1,54 @@
+package puppy
```

Comment:
IMO I see people "over-structuring" go projects quite often (i.e. too many, too fine grained packages).
I'd keep everything in a single package.
Saves you prefixing errors with `puppy.`.

Reply:
I may be misunderstanding sth here.

* Due to: https://github.com/anz-bank/go-course/tree/master/08_project I should keep storer implementations in `store` directory.
* Due to https://golang.org/ref/spec#Package_clause `go` may require keeping package files in a flat directory.
`A set of files sharing the same PackageName form the implementation of a package. An implementation may require that all source files for a package inhabit the same directory.`

Should I:
* leave packages as is?
* modify package structure?
* read about something I'm missing?

Reply:
No you are right - who came up with this lab (oh yeah, that was me....) Sorry!
Carry on